[SSL] Custom cipher suites clarifications (#22268)
cloudflare/cloudflare-docs: src/content/docs/ssl/edge-certificates/additional-options/cipher-suites/recommendations.mdx

Refer to [Customize cipher suites](/ssl/edge-certificates/additional-options/cipher-suites/customize-cipher-suites/) to learn how to specify cipher suites at zone level or per hostname.

:::caution

> When opting for [compatible](#compatible) or [modern](#modern), make sure to review the [related SSL/TLS settings](/ssl/edge-certificates/additional-options/cipher-suites/#related-ssltls-settings)[^4].

Reviewer: ```suggestion
Before opting for [compatible](#compatible) or [modern](#modern), review the [related SSL/TLS settings](/ssl/edge-certificates/additional-options/cipher-suites/#related-ssltls-settings)[^4].
```
